drivers/media/i2c/s5k2xx.c

```diff
@@ -0,0 +1,1468 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2019 Intel Corporation.
+
```

Comment:
For upstream the driver should probably be converted to CCI register access helpers.

Reply:
Possibly it will be not upstreamed, there is a chance it will be replaced with ccs in the future.